### DISCUSSION TITLE
Error in adding owners for SPAs

### DISCUSSION THREAD

**Discussion by hanan-commits-work** [iTwin/itwinjs-core#4336] [Q&A]
Greetings, I am getting an error when i try to add owners for any of the SPAs. A few months ago it was working fine even with adding a service as an owner, but now it doesnt work on human owners too. I was wondering if there has been some change (and whats the alternative) or it's an internal problem.
Thank you.

**Comment by hanan-commits-work** [comment]
Any update on this?

**Reply by tamanud** [reply]
Hi,

Can you please share some details on the error message or a screen shot of the error. 

Thanks

**Reply by hanan-commits-work** [reply]
![image](https://user-images.githubusercontent.com/89520756/192697253-66ce2dcb-89ce-4d97-8b8b-8066c83bd6a5.png)

here it is, Same error message occurs when I try to add human email.

**Reply by laurynasr** [reply]
Hi,

Only members of the same organization can be added as owners of an application. It looks like you have added the service account email previously when we had a bug that still allowed this, and now it's preventing further updates. You will need to remove the service account email from owners to proceed.

**Reply by hanan-commits-work** [reply]
Hi

The service account as an owner was needed earlier when I replaced Bentley's authorization with NoAuth and Token-server. It didn't work without it. I was wondering if there would be any issue if I no longer add service as an owner to the SPA.
I also tried adding a member of my organization to the SPA and same error was given.
